### PR TITLE
Add documentation on benchmark datasets

### PR DESCRIPTION
**Pull request by Rensvandeschoot**
This PR prepares the the documentation for adding the benchmark datasets

### CODE REVIEW

File: docs/source/intro/datasets.rst [hunk 1]
```diff
@@ -238,71 +238,62 @@ imported in ASReview.
...
-T. Hall, S. Beecham, D. Bowes, D. Gray, S. Counsell, in IEEE Transactions on Software
-Engineering, vol. 38, no. 6, pp. 1276-1304, Nov.-Dec. 2012. https://doi.org/10.1109/TSE.2011.103
+38 inclusions out of 5,782 papers
 
+.. figure:: ../../images/gifs/ptsd_recall_slow_1trial_fancy.gif
+   :alt: Recall curve for the ptsd dataset
 
-Dataset publication https://zenodo.org/record/1162952.
+*Virus Metagenomics*
+XXX
```
Comment (Rensvandeschoot) [inline docs/source/intro/datasets.rst:284]: Figure needed

Comment (Rensvandeschoot) [reply docs/source/intro/datasets.rst:284]: @SagevdBrand could you generate a .gif for this data?

File: docs/source/lab/exploration.rst [hunk 2]
```diff
@@ -1,78 +1,46 @@
...
 
 - Latent trajectories of trauma symptoms and resilience: the 3-year longitudinal prospective USPER study of Danish veterans deployed in Afghanistan
 - A Latent Growth Mixture Modeling Approach to PTSD Symptoms in Rape Victims
 - Peace and War: Trajectories of Posttraumatic Stress Disorder Symptoms Before, During, and After Military Deployment in Afghanistan
 - The relationship between course of PTSD symptoms in deployed U.S. Marines and degree of combat exposure
 - Trajectories of trauma symptoms and resilience in deployed US military service members: Prospective cohort study
 
+*Virus Metagenomics*:
+
+- XXX
```
[the remark this answers is not in the document]
Comment (Rensvandeschoot) [reply docs/source/lab/exploration.rst:41]: @SagevdBrand can you randomly pick 5 included papers?

File: docs/source/lab/exploration.rst [hunk 3]
```diff
@@ -91,58 +59,45 @@ ACE:
...
-(in red) after screening 50 papers, 27 after screening 100 papers and 32
-after 200 papers.
-
-
-
-Results
--------
+For the *Virus Metagenomics* dataset we expect you to find XX out of YY relevant
+papers after screening 50 papers, XXX after screening 100 papers
+and XXX after 200 papers
```
[the remark this answers is not in the document]
Comment (Rensvandeschoot) [reply docs/source/lab/exploration.rst:79]: @GerbrichFerdinands how did you calculate these numbers?

[the remark this answers is not in the document]
Comment (GerbrichFerdinands) [reply docs/source/lab/exploration.rst:79]: I did something like 

```
n_papers = total number of papers in the dataset
B = 50
A = B / n_papers 

# then compute the RRF@A
RRF@A = C

XX = C/100 * YY
```

Repeat for B = 200.

**Review by J535D165 — APPROVED** [review]

**Comment by J535D165** [comment]
Brilliant PR!

**Pull request merged**